package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-client",
   "description": "FeedHenry MBaaS Client",
-  "version": "0.15.0",
+  "version": "0.15.1",
   "author": "FeedHenry",
   "main": "index.js",
   "directories": {
     "lib": "lib"
   },
   "engines": {
-    "node": ">=0.8 < 0.11"
+    "node": "4.4"
```

Comment:
@wei-lee given conversations in the comments in [RHMAP-6063](https://issues.jboss.org/browse/RHMAP-6063), if your team are going to be looking at updating all of the modules to build with NodeJS 4 on Bob, maybe this is something that you could consider also (building fh-mbaas-api and dependencies on both versions)?